training/deepspeech_training/util/sample_collections.py

```diff
@@ -483,6 +500,9 @@ def samples_from_sources(sample_sources, buffering=BUFFER_SIZE, labeled=None):
     if len(sample_sources) == 0:
         raise ValueError('No files')
     if len(sample_sources) == 1:
-        return samples_from_source(sample_sources[0], buffering=buffering, labeled=labeled)
-    cols = list(map(partial(samples_from_source, buffering=buffering, labeled=labeled), sample_sources))
-    return Interleaved(*cols, key=lambda s: s.duration)
+        return samples_from_source(sample_sources[0], buffering=buffering, labeled=labeled, reverse=reverse)
+    cols = list(map(partial(samples_from_source,
+                            buffering=buffering,
+                            labeled=labeled,
+                            reverse=reverse), sample_sources))
```

Comment:
Maybe use this opportunity to simplify it a bit:
```suggestion
    cols = [samples_from_source(source,
                                buffering=buffering,
                                labeled=labeled,
                                reverse=reverse)
            for source in sample_sources]
```